src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationRunner.java

```diff
@@ -366,11 +368,19 @@ public String getScanServicePostContentType() {
         return RAPID_SCAN_CONTENT_TYPE;
     }
 
-    public File getContainerScanImage() {
-        Optional<Path> containerImageFilePath = detectConfigurationFactory.getContainerScanFilePath();
+    public File getContainerScanImage(Gson gson) throws IntegrationException, IOException, DetectUserFriendlyException {
+        Optional<String> containerImageFilePath = detectConfigurationFactory.getContainerScanFilePath();
         File containerImageFile = null;
         if (containerImageFilePath.isPresent()) {
-            containerImageFile = containerImageFilePath.get().toFile();
+            String containerImageUri = containerImageFilePath.get();
+            String targetPathName = String.join("", getDirectoryManager().getBinaryOutputDirectory().toString(), "/targetImage");
```

Comment:
Can getDirectoryManager().getBinaryOutputDirectory() be null?

Reply:
Added the check, thanks!